CI/package-deps-windows.ps1

```diff
@@ -0,0 +1,159 @@
+    }
+
+    if (!(Test-Path "${CrossDirBase}\x86" -PathType "Container")) {
+        Caught-Error "Missing cross-compiled build in ${CrossDirBase}\x86"
+    } elseif (!(Test-Path "${CrossDirBase}\x86_64" -PathType "Container")) {
+        Caught-Error "Missing cross-compiled build in ${CrossDirBase}\x86_64"
+    } elseif (!(Test-Path "${NativeDirBase}\win32" -PathType "Container")) {
+        Caught-Error "Missing native build in ${NativeDirBase}\win32"
+    } elseif (!(Test-Path "${NativeDirBase}\win64" -PathType "Container")) {
+        Caught-Error "Missing native build in ${NativeDirBase}\win64"
+    }
+
+    Remove-ItemIfExists "${DepsBuildDir}"
+    Ensure-Directory "${DepsBuildDir}\win32"
+    Ensure-Directory "${DepsBuildDir}\win64"
+    cd ..
+
+    $Packages = @(
+        [pscustomobject]@{Arch='x86'; WinArch='win32'}
+        [pscustomobject]@{Arch='x86_64'; WinArch='win64'}
+    )
+    Foreach ($Package in $Packages) {
+        if ($Package.Arch -eq "x86") {
+            $CrossDir = "${CrossDirBase}\x86"
+            $NativeDir = "${NativeDirBase}\win32"
+            $FinalDir = "${DepsBuildDir}\win32"
+        } elseif ($Package.Arch -eq "x86_64") {
+            $CrossDir = "${CrossDirBase}\x86_64"
+            $NativeDir = "${NativeDirBase}\win64"
+            $FinalDir = "${DepsBuildDir}\win64"
+        }
+
+        Write-Step "Copy $($Package.WinArch)/$($Package.Arch) files..."
+
+        # Copy cross-compiled deps first
+        Write-Step "Copy cross-compiled $($Package.WinArch)/$($Package.Arch) files..."
+        Copy-Item -Path "${CrossDir}\bin" -Destination "${FinalDir}" -Recurse
+        Copy-Item -Path "${CrossDir}\include" -Destination "${FinalDir}" -Recurse
+        Copy-Item -Path "${CrossDir}\licenses" -Destination "${FinalDir}" -Recurse
+
+        Write-Step "Clean up cross-compiled $($Package.WinArch)/$($Package.Arch) files..."
+        # Remove unneeded files before copying native-compiled deps
+        # Make sure symlinks still exist before trying to remove them
+        Remove-Item -Path "${FinalDir}\bin\libmbedtls.dll" -Force
+        Remove-Item -Path "${FinalDir}\bin\libmbedx509.dll" -Force
+        Remove-ItemIfExists "${FinalDir}\bin\libpng16-config"
+        Remove-ItemIfExists "${FinalDir}\bin\libpng-config"
+        Remove-Item -Path "${FinalDir}\bin\mbedcrypto.lib" -Force
+        Remove-Item -Path "${FinalDir}\bin\mbedtls.lib" -Force
+        Remove-Item -Path "${FinalDir}\bin\mbedx509.lib" -Force
+        Remove-Item -Path "${FinalDir}\bin\pngfix.exe" -Force
+        Remove-Item -Path "${FinalDir}\bin\png-fix-itxt.exe" -Force
+        Remove-ItemIfExists "${FinalDir}\bin\srt-ffplay"
+        Remove-Item -Path "${FinalDir}\bin\x264.def" -Force
+        Remove-Item -Path "${FinalDir}\bin\x264.exe" -Force
+        Remove-Item -Path "${FinalDir}\bin\zlib.def" -Force
+
+        # Copy native-compiled deps
+        Write-Step "Copy native-compiled $($Package.WinArch)/$($Package.Arch) files..."
```

Comment:
See above re. using array and a pipe.

Reply:
Similar to above, I'll probably put that on a list to investigate for a future cleanup effort.